File: src/Services/API/SRM.Domain/Individuals/Parameters/Services/ObstacleTypeService.cs
using Fix.Data;
using SRM.Data.Models.Individuals.Parameters;
using SRM.Domain.Parameters.Services;

namespace SRM.Domain.Individuals.Parameters.Services
{
    public class ObstacleTypeService : ParameterService<ObstacleType>, IObstacleTypeService
    {
        public ObstacleTypeService(IRepository<ObstacleType> repository) : base(repository)
        {
        }
    }
}

File: src/Web/SRM.WebSPA/ClientApp/src/app/shared/services/individuals/parameters/obstacle-type.service.ts
import { Injectable } from '@angular/core';
import 'rxjs/add/operator/map';
import { ObstacleType } from 'shared/models';
import { ConfigurationService } from 'shared/services/application/configuration.service';
import { DataService } from 'shared/services/application/data.service';
import { ParameterBaseService } from 'shared/services/base-services/parameter-base.service';
@Injectable()
export class ObstacleTypeService extends ParameterBaseService<ObstacleType> {

    constructor(public _configurationService: ConfigurationService, public service: DataService) {
        super(_configurationService,service)
        this._serviceApi = '/api/ObstacleType/';
    }
}
